modules/ROOT/pages/coverage-studio-concept.adoc

```diff
@@ -0,0 +1,73 @@
+= Using MUnit Coverage in Studio
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can run a coverage report from Anypoint Studio.
+
+The following configurations apply when you execute your MUnit tests in Studio. They don't apply when running tests from Maven in a CI/CD environment. 
+
+To run coverage reports from Maven, see xref:coverage-maven-concept.adoc[].
+
+== Enable Coverage Reports in Anypoint Studio
+
+See which flows, and which percentage of event processors in those flows the test covers by clicking on the Coverage button:
+
+image::using-munit-coverage-in-anypoint-studio.png["MUnit Coverage tab highlighted in the MUnit section"]
+
+The Overall Coverage value represents the percentage of the Mule application event processors that the MUnit test executes.
+
+Click *Generate Report* to get the report that details the flows and event processors in the Mule configuration files included in the MUnit test.
+
+image::munit-coverage-report-in-anypoint-studio.png["Files included in the report highlighted"]
+
+[%header,cols="20a,80a"]
+|===
+| Column      | Description
+| *Resource*    | Name of the Mule configuration file
+| *Containers* | Number of flows in the Mule configuration file
+| *Weight*      | Percentage of your application in a configuration file +
+                A percentage of the overall number of event processors in the documentation and the
```

Comment:
What does "event processors in the documentation" mean?

Reply:
Deleted in the newer version. 
Thanks!